### ISSUE TITLE
Guide: How to create a new quickstarter

### ISSUE THREAD

**Issue by rattermeyer** [opendevstack/opendevstack.github.io#51]
We should document the necessary steps to create a new quickstarter, so people can create and contribute new quickstarters as well as issue PR against existing ones.

**Comment by michaelsauter** [comment]
Yes! Maybe we can even have a script that creates a skeleton ...
